### PR TITLE
Umask config

### PR DESCRIPTION
**Pull request by chflo**
- Cleanup: Removed obsolete config print methods, and unecessary site-umask variable
- Added umask to jobs.py file

### CODE REVIEW

File: devel/libenkf/tests/enkf_umask_config_test.c [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright (C) 2016  Statoil ASA, Norway.
+
+   The file 'enkf_export_inactive_cells.c' is part of ERT - Ensemble based Reservoir Tool.
```
Comment (pgdr) [inline devel/libenkf/tests/enkf_umask_config_test.c]: wrong filename :)  That's why I simply write "This file is part of ERT ... "

**Comment by pgdr** [comment]
The tests don't pass on my machine, that is the `enkf_umask_config_test` is not run and therefore fails.

Possibly something I've done wrong in fetching this repo.  Any ideas?

```ctest
    Start 271: enkf_umask_config_test
Could not find executable /private/pgdr/ert/ert/build/bin/enkf_umask_config_test
Looked in the following places:
/private/pgdr/ert/ert/build/bin/enkf_umask_config_test
/private/pgdr/ert/ert/build/bin/enkf_umask_config_test
...

271: Test command:  "/private/pgdr/ert/ert/devel/test-data/Statoil/config/enkf_state_runpath/config_umask"
Unable to find executable: /private/pgdr/ert/ert/build/bin/enkf_umask_config_test
1/1 Test #271: enkf_umask_config_test ...........***Not Run   0.00 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.29 sec

The following tests FAILED:
	271 - enkf_umask_config_test (Not Run)
Errors while running CTest
```

File: devel/libenkf/tests/enkf_umask_config_test.c [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+#include <ert/enkf/enkf_main.h>
+#include <ert/enkf/run_arg.h>
+
+
+
+int main(int argc , char ** argv) {
+  enkf_main_install_SIGNALS();
+
+  const char * config_file             = argv[1];
+  int iens                             = 0;
```
Comment (pgdr) [inline devel/libenkf/tests/enkf_umask_config_test.c]: unused variable

File: devel/libenkf/tests/enkf_umask_config_test.c (hunk 1 above)
Comment (joakim-hove) [inline devel/libenkf/tests/enkf_umask_config_test.c]: > That's why I simply write "This file is part of ERT ... "

Smart!

**Comment by joakim-hove** [comment]
OK - this is good; the code itself is ready to be merged. 

But the test should be changed to run externally - without Statoil restricted data. I think you can just use this file: https://github.com/Ensembles/ert/blob/master/devel/test-data/local/simple_config/minimum_config as test data - and you are all set.

**Comment by joakim-hove** [comment]
> I think you can just use this file: https://github.com/Ensembles/ert/blob/master/devel/test-data/local/simple_config/minimum_config as test data - and you are all set.

Probably better to use a modified config file, with runpath pointing to a relative directory:
```txt
RUNPATH  simulations/run%d
```

**Pull request closed without merging**